The error log in my machine is below:
```
Using ld linker
Configured for DEBUG build (set with cmake -DCMAKE_BUILD_TYPE={release,debug,...})
CMake Warning at /home/codehz/vcpkg/downloads/tools/cmake-3.14.0-linux/cmake-3.14.0-Linux-x86_64/share/cmake-3.14/Modules/FindBoost.cmake:896 (message):
  New Boost version may have incorrect or missing dependencies and imported
  targets
Call Stack (most recent call first):
  /home/codehz/vcpkg/downloads/tools/cmake-3.14.0-linux/cmake-3.14.0-Linux-x86_64/share/cmake-3.14/Modules/FindBoost.cmake:1018 (_Boost_COMPONENT_DEPENDENCIES)
  /home/codehz/vcpkg/downloads/tools/cmake-3.14.0-linux/cmake-3.14.0-Linux-x86_64/share/cmake-3.14/Modules/FindBoost.cmake:1694 (_Boost_MISSING_DEPENDENCIES)
  /home/codehz/vcpkg/scripts/buildsystems/vcpkg.cmake:250 (_find_package)
  cmake_modules/FindBoostAlt.cmake:41 (find_package)
  /home/codehz/vcpkg/scripts/buildsystems/vcpkg.cmake:286 (_find_package)
  cmake_modules/ThirdpartyToolchain.cmake:661 (find_package)
  CMakeLists.txt:436 (include)


CMake Warning at /home/codehz/vcpkg/downloads/tools/cmake-3.14.0-linux/cmake-3.14.0-Linux-x86_64/share/cmake-3.14/Modules/FindBoost.cmake:896 (message):
  New Boost version may have incorrect or missing dependencies and imported
  targets
Call Stack (most recent call first):
  /home/codehz/vcpkg/downloads/tools/cmake-3.14.0-linux/cmake-3.14.0-Linux-x86_64/share/cmake-3.14/Modules/FindBoost.cmake:1018 (_Boost_COMPONENT_DEPENDENCIES)
  /home/codehz/vcpkg/downloads/tools/cmake-3.14.0-linux/cmake-3.14.0-Linux-x86_64/share/cmake-3.14/Modules/FindBoost.cmake:1694 (_Boost_MISSING_DEPENDENCIES)
  /home/codehz/vcpkg/scripts/buildsystems/vcpkg.cmake:250 (_find_package)
  cmake_modules/FindBoostAlt.cmake:41 (find_package)
  /home/codehz/vcpkg/scripts/buildsystems/vcpkg.cmake:286 (_find_package)
  cmake_modules/ThirdpartyToolchain.cmake:661 (find_package)
  CMakeLists.txt:436 (include)


CMake Warning at /home/codehz/vcpkg/downloads/tools/cmake-3.14.0-linux/cmake-3.14.0-Linux-x86_64/share/cmake-3.14/Modules/FindBoost.cmake:896 (message):
  New Boost version may have incorrect or missing dependencies and imported
  targets
Call Stack (most recent call first):
  /home/codehz/vcpkg/downloads/tools/cmake-3.14.0-linux/cmake-3.14.0-Linux-x86_64/share/cmake-3.14/Modules/FindBoost.cmake:1018 (_Boost_COMPONENT_DEPENDENCIES)
  /home/codehz/vcpkg/downloads/tools/cmake-3.14.0-linux/cmake-3.14.0-Linux-x86_64/share/cmake-3.14/Modules/FindBoost.cmake:1694 (_Boost_MISSING_DEPENDENCIES)
  /home/codehz/vcpkg/scripts/buildsystems/vcpkg.cmake:250 (_find_package)
  cmake_modules/FindBoostAlt.cmake:41 (find_package)
  /home/codehz/vcpkg/scripts/buildsystems/vcpkg.cmake:286 (_find_package)
  cmake_modules/ThirdpartyToolchain.cmake:661 (find_package)
  CMakeLists.txt:436 (include)


CMake Error at /home/codehz/vcpkg/downloads/tools/cmake-3.14.0-linux/cmake-3.14.0-Linux-x86_64/share/cmake-3.14/Modules/FindPackageHandleStandardArgs.cmake:137 (message):
  Could NOT find Lz4 (missing: LZ4_LIB)
Call Stack (most recent call first):
  /home/codehz/vcpkg/downloads/tools/cmake-3.14.0-linux/cmake-3.14.0-Linux-x86_64/share/cmake-3.14/Modules/FindPackageHandleStandardArgs.cmake:378 (_FPHSA_FAILURE_MESSAGE)
  cmake_modules/FindLz4.cmake:62 (find_package_handle_standard_args)
  /home/codehz/vcpkg/scripts/buildsystems/vcpkg.cmake:286 (_find_package)
  cmake_modules/ThirdpartyToolchain.cmake:192 (find_package)
  cmake_modules/ThirdpartyToolchain.cmake:1792 (resolve_dependency)
  CMakeLists.txt:436 (include)
```

seems unrelated to Flatbuffers

--- a/ports/flatbuffers/CONTROL
+++ b/ports/flatbuffers/CONTROL
@@ -1,5 +1,5 @@
 Source: flatbuffers
-Version: 1.12.0-1
+Version: 1.12.0
 Description: Memory Efficient Serialization Library
  FlatBuffers is an efficient cross platform serialization library for games and other memory constrained apps. It allows you to directly access serialized data without unpacking/parsing it first, while still having great forwards/backwards compatibility.
 Homepage:  https://google.github.io/flatbuffers/

--- a/ports/flatbuffers/CONTROL
+++ b/ports/flatbuffers/CONTROL
@@ -1,5 +1,5 @@
 Source: flatbuffers
-Version: 1.11.0-1
+Version: 1.12.0-1
 Description: Memory Efficient Serialization Library
  FlatBuffers is an efficient cross platform serialization library for games and other memory constrained apps. It allows you to directly access serialized data without unpacking/parsing it first, while still having great forwards/backwards compatibility.
 Homepage:  https://google.github.io/flatbuffers/

--- a/ports/flatbuffers/portfile.cmake
+++ b/ports/flatbuffers/portfile.cmake
@@ -1,5 +1,3 @@
-include(vcpkg_common_functions)
-
 vcpkg_check_linkage(ONLY_STATIC_LIBRARY)
 
 vcpkg_from_github(
@@ -15,7 +13,7 @@ vcpkg_from_github(
 )
 
 set(OPTIONS)
-if(VCPKG_CMAKE_SYSTEM_NAME STREQUAL "WindowsStore")
+if(VCPKG_TARGET_IS_UWP)
     list(APPEND OPTIONS -DFLATBUFFERS_BUILD_FLATC=OFF -DFLATBUFFERS_BUILD_FLATHASH=OFF)
 endif()
 
@@ -47,5 +45,4 @@ file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/debug/include)
 file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/bin ${CURRENT_PACKAGES_DIR}/debug/bin)
 
 # Handle copyright
-file(COPY ${SOURCE_PATH}/LICENSE.txt DESTINATION ${CURRENT_PACKAGES_DIR}/share/flatbuffers)
-file(RENAME ${CURRENT_PACKAGES_DIR}/share/flatbuffers/LICENSE.txt ${CURRENT_PACKAGES_DIR}/share/flatbuffers/copyright)
+file(INSTALL ${SOURCE_PATH}/LICENSE.txt DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT} RENAME copyright)

